src/renderer/widgets/AddProxyModal/model/confirm-model.ts

```diff
@@ -0,0 +1,71 @@
+import { createEvent, combine, restore } from 'effector';
+
+import { Chain, Account, ProxyType, Address } from '@shared/core';
+import { networkModel } from '@entities/network';
+import { Transaction } from '@entities/transaction';
+import { walletModel, walletUtils } from '@entities/wallet';
+
+type Input = {
+  chain: Chain;
+  account: Account;
+  signatory?: Account;
+  proxyType: ProxyType;
+  delegate: Address;
+  description: string;
+  transaction: Transaction;
+
+  oldProxyDeposit: string;
```

Comment:
Because `Fee / ProxyDeposit` are being calculated from UI, this data have to be stored
In future all fee calculations must be done in model (as instance with help of services)